start/software/software-transfer-via-htcondor-or-http.md

```diff
@@ -2,94 +2,63 @@
-	Log     = log/words.log.$(Cluster)
-	should_transfer_files = YES
-	transfer_input_files = http://stash.osgconnect.net/+username/words.tar.gz
-	when_to_transfer_output = ON_EXIT
-	Queue 5
-
-Submit the job:
-
-	$ condor_submit words.submit
-	Submitting job(s).....
-	5 job(s) submitted to cluster 14038.
-
-Once the jobs are completed, you can look at the output in the logs directory and verify that the job ran correctly:
-
-	$ cat log/words.out.14038-2
-	Ashkenazim |45 (0.44%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	BIOS       |45 (0.44%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Anaheim    |44 (0.43%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Aymara     |44 (0.43%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Arthurian  |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Anaxagoras |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Bactria    |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Alexis     |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Ariel      |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Aubrey     |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Baryshnikov|42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Bahia      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Angstrom   |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Asoka      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Alcatraz   |41 (0.40%) ++++++++++++++++++++++++++++++++++++++++++++++++++++
+For some OSG Connect users, it may be necessary or advatageous to install your own software. This guide will 
+describe steps and important considerations for transferring your installed software via the HTCondor submit file.  
 
+## Important Considerations
+
+How you transfer your software via HTCondor will depend on the size and location of the executable binary or tar archive 
+that needs to be transferred along with your jobs. Only software files <1GB should be transferred via HTCondor using the steps described below. Please see [Introduction to Data Management on OSG Connect](https://support.opensciencegrid.org/support/solutions/articles/12000002985) for more information regarding our policies for staging files (data, software, etc.) in OSG Connect.  
+
+Keep in mind that HTCondor will always transfer the script or binary that is specified as the `executable` in your submit 
+file. Thus, it is not necessary to follow the steps below if your software is the designated executable in your submit file.  
+
+The steps and considerations described below also apply to transferring data, and other files, as described in the [Transferring Data With HTCondor](https://support.opensciencegrid.org/support/solutions/articles/5000639787) guide.
+
+## Transferring Software From `/home`

```

Comment:
Use a title that states an objective, not the solution, e.g. "Transfer Data or Software Smaller than XXMB":